jvm-runtime/testdata/java/gradle/src/main/java/org/acme/Gradle.java

```diff
@@ -3,11 +3,11 @@
 import xyz.block.ftl.Export;
 import xyz.block.ftl.Verb;
 
-public class EchoVerb {
+public class Gradle {
```

Comment:
Was this a mistake?
```suggestion
public class Echo {
```

Reply:
The `ftl.toml` here has:

```toml
module = "gradle"
language = "java"
```
so I tried to make them match up.

Reply:
But it does look a bit odd 😂 I think it's because it's used for `TestGradle`.

Reply:
Ahhh yep yep!